src/tests/ftest/pool/RebuildTests.py

```diff
@@ -236,118 +201,69 @@ def test_multipool_rebuild(self):
-            reccount = self.params.get("reccount",
-                                       '/run/testparams/numrecords/*')
-            if objcount == 0:
-                reccount = 0
-
-            # which rank to write to and kill
-            rank = self.params.get("rank", '/run/testparams/ranks/*')
-
-            # how much data to write with each key
-            size = self.params.get("size", '/run/testparams/datasize/')
-
             # Putting the same data in both pools, at least for now to simplify
             # checking its correct
             saved_data = []
-            for i in range(0, objcount):
+            for _objc in range(self.objcount):
                 obj = None
-                for j in range(0, reccount):
+                for _recc in range(self.reccount):
 
                     # make some stuff up and write
-                    dkey = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
+                    dkey = ''.join(random.choice(string.ascii_uppercase + string.digits)
                                    for _ in range(5))
-                    akey = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
+                    akey = ''.join(random.choice(string.ascii_uppercase + string.digits)
                                    for _ in range(5))
-                    data = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
-                                   for _ in range(size))
-
-                    obj, tx = container1.write_an_obj(data, len(data),
-                                                         dkey, akey, obj, rank)
-                    obj, tx = container2.write_an_obj(data, len(data),
-                                                         dkey, akey, obj, rank)
-
+                    data = ''.join(random.choice(string.ascii_uppercase + string.digits)
+                                   for _ in range(self.size))
+
+                    #Used DAOS_OC_R1S_SPEC_RANK,    /* 1 replica with specified rank */
+                    obj, tx = container1.write_an_obj(data, len(data), dkey, akey, obj, self.rank,
+                                                      obj_cls=15)
+                    obj, tx = container2.write_an_obj(data, len(data), dkey, akey, obj, self.rank,
+                                                      obj_cls=15)
                     saved_data.append((obj, dkey, akey, data, tx))
 
-                    # read the data back and make sure its correct
-                    # containers
-                    data2 = container1.read_an_obj(size, dkey, akey,
-                                                   obj, tx)
+                    # read the data back and make sure its correct containers
+                    data2 = container1.read_an_obj(self.size, dkey, akey, obj, tx)
```

Comment:
(pylint-line-too-long) Line too long (82/80)

Reply:
not belong to current patch